test/requests/document_link_expectations_test.rb

```diff
@@ -20,20 +20,31 @@ def map_expectations(expectations)
     end
   end
 
+  class FakeHTTPResponse < Struct.new(:code, :body)
+  end
+
   def run_expectations(source)
     document = RubyLsp::Document.new(source)
-    RubyLsp::Requests::DocumentLink.new("file://fake/path/without_version.rb", document).run
+    js_content = File.read(File.join(TEST_FIXTURES_DIR, "rails_search_index.js"))
```

Comment:
I think we always automatically read every file under `test/fixtures`. Maybe we can change it to read only ruby files?